components/coding/ProjectsSection.tsx

```diff
@@ -30,7 +31,9 @@ export default function ProjectsSection({ user }: ProjectsSectionProps) {
         <div>Loading...</div>
       ) : userProjects.length === 0 ? (
         <div className="col-span-3 p-8 text-center shadow-md bg-slate-300 dark:bg-slate-900">
-          No Active Projects
+          <ExpandableContainer open title={"No Active Projects"}>
```

Comment:
mentioned this on slack but let's not commit any changes to this component since it's being used on the current profile